modules/virt-example-slb-secondary-bond.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-example-slb-secondary-bond_{context}"]
+= Example: Source load balancing (SLB) on a secondary bond interface
+
+You can run the Open vSwitch (OVS) `balance-slb` mode on a secondary bond interface so to isolate traffic from the primary network. This configuration for managing traffic provides your cluster with high availability and traffic segmentation capabilities. 
+
+The following YAML file shows an example of a manifest for a secondary bond interface. The file includes samples values that you must replace with your own information.
+
+[source,yaml]
+----
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: bond1-policy <1>
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: <node01>
+  desiredState:
+    interfaces:
+    - name: br1
+      type: ovs-interface <2>
+      state: up
+      copy-mac-from: enp2s0 <3>
+      ipv4:
+        enabled: true
+        dhcp: true
+    - name: br1
+      type: ovs-bridge
+      state: up
+      ipv4:
+        enabled: false
+        dhcp: false
+      bridge:
+        port:
+          - name: br1 <4>
+          - name: bond0
+            link-aggregation:
+              mode: balance-slb <5>
+              port: <6>
+                - name: enp2s0
+                - name: enp3s0
+    - name: enp2s0
+      type: ethernet
+      state: up
+      ipv4:
+        enabled: false
+      ipv6:
+        enabled: false
+    - name: enp3s0
+      type: ethernet
+      state: up
```

Comment:
```
    mtu: 1500  <1> 

<1> `br1` MTU as set by DHCP must be manually set on the bond ports 
```


The DHCP MTU settings will not propagate down to the bond ports.  If we set the MTU using DHCP we have to manually set the MTU on the bond ports to match.

Reply:
:heavy_check_mark: